plugin/sampling/strategystore/adaptive/aggregator.go

```diff
@@ -18,16 +18,15 @@ import (
 	"sync"
 	"time"
 
-	"github.com/uber/jaeger-client-go"
-
 	"github.com/jaegertracing/jaeger/cmd/collector/app/sampling/model"
 	"github.com/jaegertracing/jaeger/cmd/collector/app/sampling/strategystore"
 	"github.com/jaegertracing/jaeger/pkg/metrics"
 	"github.com/jaegertracing/jaeger/storage/samplingstore"
 )
 
 const (
-	maxProbabilities = 10
+	maxProbabilities         = 10
+	SamplerTypeProbabilistic = "probabilistic"
```

Comment:
Does this need to be exported?

Reply:
As they can be found in several places, maybe they could be moved in an internal package dedicated to those constants and they won't be public but only dedicated to jaeger.

Reply:
Thanks. into it.